app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -89,7 +106,7 @@ export default function ResourceAuthoringPage() {
         <Grid.Col span={6}>
           <Paper withBorder>
             <Prism language="json" colorScheme="light">
-              {resourceQuery.data ? JSON.stringify(resourceQuery.data, null, 2) : ''}
+              {resource ? JSON.stringify(resource, null, 2) : ''}
```

Comment:
This is now huge for real resources. I think we might want to limit the height of the JSON view and make it scrollable, essentially allowing a user to scroll in the JSON while keeping the inputs on the left side where they are